drivers/ethernet/Kconfig

```diff
@@ -59,8 +59,6 @@ source "drivers/ethernet/Kconfig.cyclonev"
 
```

Comment:
Could you add more description for the commit `drivers: ethernet: Fix hanging configuration option`, the commit message is not descriptive enough.